pkg/rainbow/storage/dbram/dbram.go

```diff
@@ -12,14 +12,24 @@ import (
 
 // DB client.
 type DB struct {
-	options []rainbow.Option
+	options  []rainbow.Option
+	cpFormat rainbow.CPFormat
 }
 
 func (db *DB) InsertOptions(options []rainbow.Option) error {
 	db.options = options
 	return nil
 }
 
+func (db *DB) InsertCPFormat(table rainbow.CPFormat) error {
+	db.cpFormat = table
+	return nil
+}
+
 func (db *DB) GetAllOptions() ([]rainbow.Option, error) {
 	return db.options, nil
 }
+
+func (db *DB) GetCPFormat() (rainbow.CPFormat, error) {
+	return db.cpFormat, nil
+}
```

Comment:
Do we still need this methods ? I thought we said we do not store `CPFormat` in the database.